new-frontiers-14/frontier-station-14
# Discord Ahelp Reply System

## About the PR


This adds the Server sided Code for the AHelp System to be able to reply from discord to the ingame ahelp system. 
I will be marking this as a draft for now due to the bot not finished yet to handle this system and to let in feedback.
This is my first big PR with c#, so i would love to get some feedback about bad stuff.

## Why / Balance

Allows Admins to reply to Ahelps from discord, not requiring them to get ingame.

## How to test

_oh boi_
1. Fork and run server 
2. Make sure Status api is enabled. (status.enabled)
3. Make sure an API token is set (admin.api_token)
4. Make a POST request to _inser_server_ip_/admin/actions/send_bwoink with the following content:
`{
 "Text": "Your Message Here",
 "Guid": "Your_Guid_Here"
}`
Set Authoriazion Header to `SS14Token apitoken`
_[A (nearly) Pasteready command can be found here](https://haste.skyblockcommunity.com/ecasifuwuj.sh)_
5. See getting an Ahelp Message with your content

## Media

_This was tested on a online server and fully works._
![image](https://github.com/user-attachments/assets/7a85b9ca-3eb3-443f-b27d-7917049f70c8)


## Requirements



## Breaking changes


**Changelog**



_I do not know if i can make an admin only changelog here?
Nothing really for the user._
